go/app/main.go

```diff
@@ -51,6 +122,34 @@ func getImg(c echo.Context) error {
 	return c.File(imgPath)
 }
 
+func getAllItems(c echo.Context) error {
+	items := dataJson()
+	return c.JSON(http.StatusOK, items)
+}
+
+func getItem(c echo.Context) error {
+	items := dataJson()
+
+	//Get the parameter
+	idParm := c.Param("id")
+	id, err := strconv.Atoi(idParm)
+
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	//Search for the id
+	SelectedItem := Item{}
+	for i := 0; i < len(items.Items); i++ {
```

Comment:
nit: you can make this loop more readable by using Go's `for` and `range` loop pattern.